envs/env-py310.yml

```diff
@@ -193,7 +196,7 @@ dependencies:
   - kkcalc
   - ppmac
   - pychx >=4.1.2
-  - xpdacq >=1.1.4
+  - xpdacq ==1.0.0
```

Comment:
Intentionally downgraded?

Reply:
Yes, the latest version of `xpdacq` does not work with the latest pdfstream. It was discussed with the beamline staff.

Reply:
Discussed it again with the BL staff, and we need a bit older version of `pdfstream` to work correctly in tandem. This was addressed via https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/24/commits/1fda94abda2ad8ee2b8cf17484ad892d2fa86fd7.